cnf-tests/testsuites/e2esuite/multinetworkpolicy/multinetworkpolicy_sriov.go

```diff
@@ -0,0 +1,819 @@
+			eventually30s(nsZ_podB).Should(Reach(nsX_podA, OnPort(port5555)))
+
+			eventually30s(nsX_podC).Should(Reach(nsX_podA, OnPort(port6666)))
+			eventually30s(nsY_podC).Should(Reach(nsX_podA, OnPort(port6666)))
+			eventually30s(nsZ_podC).Should(Reach(nsX_podA, OnPort(port6666)))
+
+			// Not allowed
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666)))
+			eventually30s(nsY_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666)))
+			eventually30s(nsZ_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666)))
+
+			eventually30s(nsX_podC).ShouldNot(Reach(nsX_podA, OnPort(port5555)))
+			eventually30s(nsY_podC).ShouldNot(Reach(nsX_podA, OnPort(port5555)))
+			eventually30s(nsZ_podC).ShouldNot(Reach(nsX_podA, OnPort(port5555)))
+		})
+	})
+
+	Context("Ports/Protocol", func() {
+		It("Allow access only to a specific port/protocol TCP", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					Ports: []multinetpolicyv1.MultiNetworkPolicyPort{{
+						Port:     &port5555, // Default protocol: TCP
+						Protocol: &protoTCP,
+					}},
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{{
+						PodSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"pod": "b",
+							},
+						},
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsX_podB).Should(Reach(nsX_podA, OnPort(port5555), ViaTCP))
+
+			// Not allowed
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaTCP))
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaUDP))
+
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port5555), ViaUDP))
+
+			if sctpEnabled {
+				eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaSCTP))
+				eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port5555), ViaSCTP))
+			}
+		})
+
+		It("Allow access only to a specific port/protocol UDP", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					Ports: []multinetpolicyv1.MultiNetworkPolicyPort{{
+						Port:     &port6666,
+						Protocol: &protoUDP,
+					}},
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{{
+						PodSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"pod": "b",
+							},
+						},
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsX_podB).Should(Reach(nsX_podA, OnPort(port6666), ViaUDP))
+
+			// Not allowed
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaTCP))
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA, OnPort(port6666), ViaSCTP))
```

Comment:
We should check that SCTP is enabled like in line 544

Reply:
Yes, that's better.